test/extended/cli/explain.go

```diff
@@ -375,11 +359,11 @@ var _ = g.Describe("[cli] oc explain networking types", func() {
 
 	oc := exutil.NewCLI("oc-explain", exutil.KubeConfigPath())
 
-	g.It("should contain proper fields description for special networking types", func() {
-		for _, st := range specialNetworkingTypes {
-			e2e.Logf("Checking %s, Field=%s...", st.gv, st.field)
-			o.Expect(verifyExplain(oc, nil, schema.GroupVersionResource{},
-				st.pattern, st.field, fmt.Sprintf("--api-version=%s", st.gv))).NotTo(o.HaveOccurred())
+	g.It("should contain proper spec+status for networking CRDs", func() {
```

Comment:
If networking is now following the usual types, I'd expect it to be part of `crdTypes` as the others are. No exceptions.

Reply:
Yes, but we need to split the test out since these crds are applicable only for Openshift SDN and not OVN.

Reply:
:+1: